Add blog review guideline to Reviewing PRs page (#48548)
kubernetes/website: content/en/docs/contribute/review/reviewing-prs.md

- Do the changes show up in the Netlify preview? Be particularly vigilant about lists, code
  blocks, tables, notes and images.

### Blog

> - Early feedback on blog posts is welcome via a Google Doc or HackMD. Please request input early from the [#sig-docs-blog Slack channel](https://kubernetes.slack.com/archives/CJDHVD54J).

Reviewer: ```suggestion
- Early feedback on blog posts is welcome via a Google Doc or HackMD. We encourage contributors
  to request input early from the [#sig-docs-blog Slack channel](https://kubernetes.slack.com/archives/CJDHVD54J), but we can also run the whole write-commit-review-update cycle within GitHub. Using a
  Google or HackMD document makes the feedback loop shorter, but does mean there is a point where
  the work needs to shift over to Git(Hub).
```

- Before reviewing blog PRs, be familiar with [Submitting blog posts and case studies](/docs/contribute/new-content/blogs-case-studies/).
- We are willing to mirror any blog article that was published to https://kubernetes.dev/blog/ (the contributor blog) provided that:
- the mirrored article has the same publication date as the original (it should have the same publication time too, but you can also set a time stamp up to 12 hours later for special cases)

>   - for PRs that arrive the original article was merged to https://kubernetes.dev/, there haven't been 

Reviewer: ```suggestion
  - [for PRs that arrive _after_ the original article was merged to https://kubernetes.dev/], there haven't been
```

Reviewer: You could use parens / round brackets instead. In the UK we might more commonly use square brackets, to mean "the sentence also makes sense if you omit this part".

Reviewer 2: Works for me—I hadn't realized there was a difference between the way we use parens/brackets between here and there. Good to know.

  This consideration applies to mirrored articles too; although we are willing to consider all valid
  contributor articles for mirroring if a PR is opened, we don't mirror all of them.

- We only mark blog articles as maintained (`evergreen: true` in front matter) if the Kubernetes project 
  is happy to commit to maintaining them indefinitely. Some blog articles absolutely merit this, and we 
  always mark our release announcements evergreen. Check with other contributors if you are not sure 
  how to review on this point.
- The [content guide](/docs/contribute/style/content-guide/) applies unconditionally to blog articles
  and the PRs that add them. Bear in mind that some restrictions in the guide state that they are only relevant to documentation; those restrictions don't apply to blog articles.
- The [style guide](/docs/contribute/style/style-guide/) largely also applies to blog PRs, but we make some exceptions.
  
  - it is OK to use “we“ in a blog article that has multiple authors, or where the article introduction clearly indicates that the author is writing on behalf of a specific group.

>   - we avoid using Kubernetes shortcodes for callouts (such as `{{< caution >}}`)

Reviewer 3: The build preview has failed with the following error [(logs here)](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/671c22ce2f61a700081eba7e):
 
> Error: error building site: process: readAndProcessContent: "/opt/build/repo/content/en/docs/contribute/review/reviewing-prs.md:197:65": failed to extract shortcode: shortcode "caution" must be closed or self-closed

Below is a suggestion that will hopefully help to resolve the error.

```suggestion
  - we avoid using Kubernetes shortcodes for callouts (such as `{{</* caution */>}}`)
```

  - statements about the future are OK, albeit we use them with care in official announcements on
    behalf of Kubernetes

>   - code samples don't need to use the `{{< code_sample >}}` shortcode, and often it is better if they do not

Reviewer 3: The build preview has failed again with the following error [(logs here)](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/671e7d918ede570008dba59f):

> ERROR [en] "" not found in "docs/contribute/review"

Similar to the previous error. Suggested fix below.
_(In the final rendered page, it will display as intended,`{{< code_sample >}}`)._

```suggestion
  - code samples don't need to use the `{{</* code_sample */>}}` shortcode, and often it is better if they do not
```
